package.json

```diff
@@ -22,20 +22,23 @@
   ],
   "dependencies": {
     "apollo-client": "^1.4.2",
-    "babel-runtime": "^6.23.0"
+    "babel-preset-env": "^1.5.2",
```

Comment:
But thats how it's meant to work. So that when you create an app bundle, apollo-upload-client does include baked-in helper code, duplicated by other dependencies – they get shared between modules and only appear once in your final Webpack bundle. Also, babel does not run on node_modules, only the current module.